apache/superset
# chore: :globe_with_meridians: Translations added to ES .po file

> ### SUMMARY
> Update spanish translations for time range filter options
> 
> ### TESTING INSTRUCTIONS
> Change language settings to Spanish and check translations via GUI or manually check them in messages.po file.
> 
> ### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
>
> Before: 
>
> ![image](https://github.com/user-attachments/assets/115ec441-bc9a-4e7f-b60e-e392ee6cd3a8)
>
> After: 
>
> ![image](https://github.com/user-attachments/assets/4c702a22-c189-4e47-aee0-3baa9f1c4330)
>
> ### ADDITIONAL INFORMATION
> * [ ]  Has associated issue:
> * [ ]  Required feature flags:
> * [x]  Changes UI
> * [ ]  Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
>   
>   * [ ]  Migration is atomic, supports rollback & is backwards-compatible
>   * [ ]  Confirm DB migration upgrade and downgrade tested
>   * [ ]  Runtime estimates and downtime expectations provided
> * [ ]  Introduces new feature or API
> * [ ]  Removes existing feature or API
